recipe/meta.yaml

```
@@ -11,7 +11,7 @@ package:
   version: {{ version }}
 
 build:
-  number: 1
+  number: 0
```

We need to increase more aggressively here as we build packages for dev and main versions and in this PR, only main, not dev has a version change.
```suggestion
  number: 2
```